### DISCUSSION TITLE
Back End

### DISCUSSION THREAD

**Discussion by jhonymiler** [dev-pira/site#149] [Ideas]
Sei que é usado Firebase como graphql, mas talvez com a adição de cadastros ou o próprio crescimento do site, não seria interessante já ter um back end própdio? Como tem muitos aqui que trabalham com javascript, sujiro o backend do AdonisJS, ele é uma arquitetura excelente para construir o back end.

**Comment by murilobeltrame** [comment]
Por hora a gente tah entendendo suficiente o nosso backend atual.
Alias, nosso backend está rodando no Contentful - tem até uma interface administrativa legalzinha.
Por enquanto o que a gente tem de recurso e o que estamos pensando pro futuro próximo, parece estar ok ... e ainda economiza com infra :P
